src/methods/SANA.cpp

```diff
@@ -315,7 +315,7 @@ Alignment SANA::run() {
             align = hillClimbingAlignment(align, (long long int)(10000000)); //arbitrarily chosen, probably too big.
             cout << hill.elapsedString() << endl;
         }
-#define PRINT_CORES 0
+#define PRINT_CORES 1
```

Comment:
Leave PRINT_CORES 0 by default, and add code instead to put these scores (unweighted only for now) to a ".scores" file.